.github/workflows/release.yml

```diff
@@ -228,7 +228,8 @@ jobs:
 
   windows-release-wheel:
     name: Build release wheels for Windows
-    runs-on: windows-latest
+    runs-on: windows-2019
+    timeout-minutes: 600
```

Comment:
I think we can revert this change, since the maximum runtime on the default GitHub actions runners is [6h by default anyway](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits).
```suggestion
```